hubconf.py

```diff
@@ -108,3 +108,32 @@ def umx(
         unmix.eval()
 
     return unmix.to(device)
+
+
+def separator(
+    targets=['vocals', 'drums', 'bass', 'other'],
+    model_name='umxhq',
+    device='cpu', *args, **kwargs
+):
+    """
+    Create a complete Separator exploiting 2-channel/stereo BiLSTM UMX with
+    several targets, trained on MUSDB18-HQ or MUSDB18.
+
+    Args:
+        targets (str): select the targets for the source to be separated.
+                       a list including: ['vocals', 'drums', 'bass', 'other'].
+                       If you don't pick them all, you probably want to
+                       activate the `residual_model=True` option.
+        pretrained (bool): If True, returns a model pre-trained on MUSDB18-HQ
+        device (str): selects device to be used for inference
+    """
+    from filtering import Separator
+
+    model = Separator(
+        targets=targets,
+        model_name=model_name,
+        device=device,
+        *args, **kwargs
+    )
+
+    return model.to(device)
```

Comment:
👍

Reply:
oh, actually I forgot to change this for latest version, TODO

Reply:
done